entrypoint.sh

```diff
@@ -21,9 +21,11 @@ set -e
 
 echo "First Install";
 ls -alh
-echo "Install Yarn"
-[ -f yarn.lock ] && yarn install --frozen-lockfile --prefer-offline
-[ -f package-lock.json ] && npm install
+if [ "$2" = true ] ; then
```

Comment:
Would just checking if `node_modules` directory be enough here instead?  Also, perhaps switching to the `npm ci` command would be enough anyway?

```suggestion
if [ ! -d "./node_modules" ] || [ "$2" = true ] ; then
```

Reply:
> Would just checking if node_modules directory be enough here instead?

Yeah, good idea 🙂

> perhaps switching to the npm ci command would be enough anyway?

In our case it's not because of `npm ci`, but because lerna is doing a lot of things and for some reason, it was failing inside the docker container. I think it was due to permission problems. I usually had to use something like this when we used docker: https://github.com/nodejs/docker-node/issues/740

So I'd leave it with `npm install` but allow people to do their own installation in another step.

Reply:
After reading a bit, it seems like `npm ci` is a better choice, as it won't update the `package-lock.json` file 👍